thread/src/components/LoginForm.js

```diff
@@ -1,57 +1,82 @@
 import React, { Component } from 'react';
 import { Container, Content, Form, Item, Input, Label, Footer, FooterTab, Button, Text, Spinner } from 'native-base';
 import { connect } from 'react-redux';
+import {  View, StyleSheet } from 'react-native';
 import { Actions } from 'react-native-router-flux';
 import * as actions from '../actions';
-import { LoadingButton } from './Common';
+import styled from 'styled-components';
+import Logo from './Logo';
+import {LoadingButton, CommonContainer, CommonField} from './Common';
+
+const TermsButtonStyle = styled(Text)`
+  font-size:14px;
+  text-align:center;
+  color:white;
+
+  margin-top:40px;
+`;
+
 
 class LoginForm extends Component {
-    
-    onSubmitPress = () => {
+
+  onSubmitPress = () => {
         const { email, password } = this.props;
 
         this.props.loginUser({ email, password });
     }
 
     render() {
         return (
-            <Container>
-                <Content>
-                    <Form>
-                        <Item stackedLabel>
-                            <Label>Email</Label>
-                            <Input
-                                onChangeText={this.props.loginEmailChanged}
-                                value={this.props.email}
-                            />
-                        </Item>
-                        <Item stackedLabel last>
-                            <Label>Password</Label>
-                            <Input
-                                secureTextEntry
-                                onChangeText={this.props.loginPasswordChanged}
-                                value={this.props.password}
-                            />
-                        </Item>
+
+            <CommonContainer>
+              <Content style={{padding:20}}>
+                <Logo />
+                  <Form>
+                      <CommonField
+                          onChangeText={this.props.loginEmailChanged}
+                          value={this.props.email}
+                          style={{marginTop: 30,  marginLeft: -5 }}
+                          label="Username"
+                          />
+                        <CommonField
+                          onChangeText={this.props.loginPasswordChanged}
+                          value={this.props.password}
+                          style={{marginTop: 30, marginLeft: -5}}
+                          secureTextEntry = {true}
```

Comment:
for future reference 

```
<Component  secureTextEntry /> === <Component  secureTextEntry={true} />
```

Reply:
I tried that initially and it didn't work

Reply:
ok, good with me then